seahorn/jobs/hash_table_init_unbounded/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+add_executable(hash_table_init_unbounded
+  aws_hash_table_init_unbounded_harness.c)
+set(MAX_TABLE_SIZE 8)
+math(EXPR MEMCPY_UNROLL_BND "${MAX_TABLE_SIZE} * 64")
```

Comment:
maybe this is a copy-paste error? the name of the task is `unbounded`, so why do we bound?